### ISSUE TITLE
Popup Issue When Accessibility Setting Set to High Contrast Issue 43283

### ISSUE THREAD

**Issue by sdwjoomla** [joomla-projects/guided-tours#19]
Issue Tracker link:  https://github.com/joomla/joomla-cms/issues/43283

**Comment by garstud** [comment]
## Steps to reproduce the issue
Activate accessibility in your user profile by setting "Hight contrast to Yes.
Save your account
Launch a guided tour

## Expected result
The tour will work

## Actual result
The tour is blocked !

## System information (as much as possible)

## Additional comments
Accessibility add a layer in the body

Why behavior change ?  
identical issue when trying to target an object that needs to scroll the page (pagination block, ... )

**Comment by obuisard** [comment]
When using accessibility features, a filter is added to the body tag.
Maybe a solution would be to make sure Shepherd gets a different parent than &lt;body&gt;

**Comment by sdwjoomla** [comment]
@garstud @obuisard with the update of Shephard is this still blocked?

**Comment by obuisard** [comment]
> @garstud @obuisard with the update of Shephard is this still blocked? 

Not sure if it solves it, @garstud ?

**Comment by garstud** [comment]
Well @obuisard , surprisingly, ... I just tested it in "**High Contrast**" user mode ... I can't trigger the bug on a Joomla patched with the latest shepherd.js 14.5.0!?
The tour is not blocked anymore!?

Hmmmm, it seems too good to be true :D ... I'll run additional checks this week!

CC @sdwjoomla 

![Image](https://github.com/user-attachments/assets/c25dc4a9-3e55-44cb-b048-2b241d577b5b)

![Image](https://github.com/user-attachments/assets/33afa633-f1a2-42b6-8fb6-33ce8de5ab39)

**Comment by garstud** [comment]
Hummm ... after additionnals test on the GT "Create articles" ... No blocking problems in Tour navigation, but a few minor issues: Some fields aren't working properly...

- The category list is accessible ( (step 7/14)
- the Feature radio  **is not accessible** (step 8/14)
- the access level list **is not accessible** (step 9/14)
- the Tags list is accessible ( (step 10/14)
- the Note input is accessible ( (step 11/14)
- the Version Note input is accessible (step 12/14)
- the Save é Close buton is working (step 13/14)

Maybe it depends on how we target fields for GT steps?

CC @obuisard @sdwjoomla

**Comment by obuisard** [comment]
> Well [@obuisard](https://github.com/obuisard) , surprisingly, ... I just tested it in "**High Contrast**" user mode ... I can't trigger the bug on a Joomla patched with the latest shepherd.js 14.5.0!? The tour is not blocked anymore!?

Tours were not really blocked, the popup would just no longer be showing properly (not being placed properly on the screen).
Also, make sure you have a filter set on the body tag.

**Comment by obuisard** [comment]
> Hummm ... after additionnals test on the GT "Create articles" ... No blocking problems in Tour navigation, but a few minor issues: Some fields aren't working properly...
> 
> * The category list is accessible ( (step 7/14)
> * the Feature radio  **is not accessible** (step 8/14)
> * the access level list **is not accessible** (step 9/14)
> * the Tags list is accessible ( (step 10/14)
> * the Note input is accessible ( (step 11/14)
> * the Version Note input is accessible (step 12/14)
> * the Save é Close buton is working (step 13/14)
> 
> Maybe it depends on how we target fields for GT steps?

Do you mean 'not accessible' as not being usable with arrow keys? 
- Radio buttons: we know they are having issues
- Access level: same kind of element as 'status'. Do you have the same issue for 'status'? You did not mention it.

**Comment by garstud** [comment]
> Do you mean 'not accessible' as not being usable with arrow keys?
> 
> * Radio buttons: we know they are having issues
> * Access level: same kind of element as 'status'. Do you have the same issue for 'status'? You did not mention it.

@obuisard , well,  '**not accessible**' = it is impossible to click on the field and to change or input a value ... but the Shepherd modal works well on these steps (showing properly and placed in a correct position!).  
**Status list** step is working correctly, no problem with the list field and the Shepherd popup!

I have to analyze the **hint target**(?) of the **access list step** , because it should works normally with Shepherd v14.5, i think!? (like Status list or Category list, ...)

**Comment by garstud** [comment]
Related to #49
